ffi/capi/src/decimal.rs

```diff
@@ -77,7 +70,7 @@ pub mod ffi {
             secondary_group_size: u8,
             min_group_size: u8,
             digits: &[DiplomatChar],
-            grouping_strategy: FixedDecimalGroupingStrategy,
+            grouping_strategy: Option<FixedDecimalGroupingStrategy>,
```

Comment:
I think it's inconsistent to not have an options bag on some APIs, when the Rust API takes an options bag. Everywhere you call `Options::default()` in a constructor it should really take a bag.

Reply:
Yeah, perhaps we should tweak this